attestation/context.go

```diff
@@ -32,9 +32,11 @@ type ErrInvalidOption struct {
 type RunType string
 
 const (
-	Internal    RunType = "internal"
```

Comment:
Internal being a special run type always kind of felt bad, imo, and having 3 attestors treated specially made for hard to reason about edge cases when configuring these attestors through the CLI.

The special edge cases around these 3 attestors I believe are covered by creating the material and product run stages, though they honestly probably could just live inthe pre and post stages. The other things that made these three special are somewhat witness CLI specific, as other use cases of go-witness may not be concerned about files or commands at all (consider approval attestations, etc), so those cases have been moved to the CLI code base.